django/contrib/gis/db/backends/postgis/schema.py

```diff
@@ -50,6 +82,9 @@ def _create_index_sql(self, model, *, fields=None, **kwargs):
             expressions=expressions,
         )
 
+    def _create_spatial_index_name(self, model, field):
+        return self._create_index_name(model._meta.db_table, [field.column], "_id")
```

Comment:
This hook is unnecessary as it's a single method call.

Reply:
Indeed but we need to create names in two places and it makes this consistent with the Oracle and MySQL backends which also have `_create_spatial_index_name` hooks. Happy to remove/inline if you insist though.

Reply:
We cannot mix unnecessary refactoring with a bugfix as this will make it more difficult to fix any potential regression introduced by this code.